docs/spec/src/integration/spec/5-lifecycle-phases.md

```diff
@@ -24,30 +17,28 @@ EigenDA *Proxy* is used as the main entrypoint for secure dispersal and retrieva
 
 
 ### System Flow
-1. *Proxy* takes a raw [payload](./3-data-structs.md#payload) bytes on `/put` endpoint and [converts](#payload-to-blob-encoding) it into a [blob](./3-data-structs.md#blob).
-
-1. Using `latest_block_number` (lbn) number fetched from ETH RPC node, *Proxy* calls the router to get the `EigenDACertVerifier` [contract](./4-contracts.md#eigendacertverifier) address *most likely* (if using [`EigenDACertVerifierRouter`](./4-contracts.md#eigendacertverifierrouter)) to be committed to by the `reference_block_number` (rbn) returned by the EigenDA disperser.
+1. *EigenDA Client* takes a raw [payload](./3-data-structs.md#payload) bytes and [converts](#payload-to-blob-encoding) it into a [blob](./3-data-structs.md#blob).
```

Comment:
You have `1.` used twice in this list

Reply:
9d35a2438b9d4e928bbab9d95f52d55815294cc7